app/src/test/java/com/gholem/moneylab/features/add/viewmodel/AddTransactionViewModelTest.kt

```diff
@@ -1,54 +1,164 @@
 package com.gholem.moneylab.features.add.viewmodel
```

Comment:
I couldn't check this file, because I don't have experience with mockito and navigation and this looks complicated xd

Reply:
Short info from my teammate about this file (in Polish, but I can translate it if neccessary):

"1. Struktura testów w porządku (podział na trzy segmenty), ale komentarze są niepotrzebne i zaciemniają logikę
2. Mockito w tej formie w pierwszych dwóch testach nic nie weryfikuje. Żeby zweryfikować wysołanie na mocku używamy funkcji verify(jakisMock).cosZrobil
3. Osobny Rule do przypięcia TestDispatchera jest spoko pomysłem, jeżeli testów będzie więcej. Dla jednego testu to by była przesada.
4. Castowanie czwartego mocka wydaje się niepotrzebne
5. Te wszystkie użycia pól z AddTransactionViewModel.Action najlepiej będzie zaimportować, bezpośrednio, będzie się lepiej czytało testy
6. Ogólnie polecam się bardziej skupić na czytelności testów dla człowieka, bo technicznie są w porządku, ale nie mają formy którą można wygodnie czytać, a nazwy nic nie mówią. Testy są najpierw dla innych programistów, zamiast dokumentacji, a dopiero potem dla frameworku który je uruchamia."